src/Glpi/Form/Destination/CommonITILField/ITILActorField.php

```diff
@@ -93,6 +98,14 @@ public function renderConfigForm(
                 'input_name'      => $input_name . "[" . ITILActorFieldConfig::SPECIFIC_QUESTION_IDS . "]",
                 'possible_values' => $this->getITILActorQuestionsValuesForDropdown($form),
             ],
+
+            // Specific additional config for USER_FROM_OBJECT_ANSWER and GROUP_FROM_OBJECT_ANSWER strategy
+            'group_object_answer_extra_field' => [
```

Comment:
If it applies to users too, the name should not contain `group`.

```suggestion
            'object_answer_extra_field' => [
```

Reply:
👍 don't forget to update references